moped-database/seeds/1602292389297_initial_seed_staging.sql

```diff
@@ -16,6 +16,55 @@ SET xmloption = content;
+INSERT INTO public.feature_street_segments (id, component_id, name, knack_id, ctn_segment_id, project_extent_id, from_address_min, render_type, to_address_max, full_street_name, line_type, symbol, source_layer, geography, is_deleted) VALUES (7, 2, NULL, NULL, 136700, NULL, 200, NULL, 217, 'CHALMERS AVE', 'On-Street', 2, 'ATD_ADMIN.CTN', '0105000020E61000000200000001020000000200000092E9D0E9796E58C0C05AB56B42423E40DD990986736E58C0CE35CCD078423E4001020000000200000032C7F2AE7A6E58C09A97C3EE3B423E40B284B531766E58C05797530262423E40', false);
+INSERT INTO public.feature_street_segments (id, component_id, name, knack_id, ctn_segment_id, project_extent_id, from_address_min, render_type, to_address_max, full_street_name, line_type, symbol, source_layer, geography, is_deleted) VALUES (8, 2, NULL, NULL, 100181, NULL, 1700, NULL, 1724, 'E 3RD ST', 'On-Street', 2, 'ATD_ADMIN.CTN', '0105000020E610000001000000010200000002000000DD990986736E58C0CE35CCD078423E40200890A1636E58C0FD4B529962423E40', false);
+INSERT INTO public.feature_street_segments (id, component_id, name, knack_id, ctn_segment_id, project_extent_id, from_address_min, render_type, to_address_max, full_street_name, line_type, symbol, source_layer, geography, is_deleted) VALUES (9, 2, NULL, NULL, 131873, NULL, 1800, NULL, 1824, 'E 3RD ST', 'On-Street', 2, 'ATD_ADMIN.CTN', '0105000020E610000002000000010200000002000000200890A1636E58C0FD4B529962423E40A699EE75526E58C0ECDADE6E49423E40010200000002000000747CB438636E58C05797530262423E40A699EE75526E58C0ECDADE6E49423E40', false);
+INSERT INTO public.feature_street_segments (id, component_id, name, knack_id, ctn_segment_id, project_extent_id, from_address_min, render_type, to_address_max, full_street_name, line_type, symbol, source_layer, geography, is_deleted) VALUES (10, 2, NULL, NULL, 153152, NULL, 300, NULL, 322, 'CHICON ST', 'On-Street', 2, 'ATD_ADMIN.CTN', '0105000020E610000003000000010200000002000000A699EE75526E58C0ECDADE6E49423E40E5B512BA4B6E58C04B74965984423E400102000000020000005A4A9693506E58C0C09481035A423E40E5B512BA4B6E58C04B74965984423E40010200000002000000A699EE75526E58C0ECDADE6E49423E4090BFB4A84F6E58C05797530262423E40', false);
+INSERT INTO public.feature_street_segments (id, component_id, name, knack_id, ctn_segment_id, project_extent_id, from_address_min, render_type, to_address_max, full_street_name, line_type, symbol, source_layer, geography, is_deleted) VALUES (11, 2, NULL, NULL, 147368, NULL, 400, NULL, 416, 'CHICON ST', 'On-Street', 2, 'ATD_ADMIN.CTN', '0105000020E610000001000000010200000002000000E5B512BA4B6E58C04B74965984423E40DD0BCC0A456E58C06EDBF7A8BF423E40', false);
+
```

Comment:
> `0105000020E610000001000000010200000002000000E5B512BA4B6E58C04B74965984423E40DD0BCC0A456E58C06EDBF7A8BF423E40`

i believe this is the [well-known-binary (WKB) format](https://postgis.net/docs/using_postgis_dbmanagement.html#OpenGISWKBWKT)